Sources/Orbit/Components/InputField.swift

```diff
@@ -96,6 +120,15 @@ public struct InputField: View {
                 }
         }
     }
+
+    @ViewBuilder var securedSuffix: some View {
+        Icon.Content.icon(isSecureTextEntry ? .visibility : .visibilityOff, size: .default)
+        .view()
+        .padding(.horizontal, .xSmall)
+        .onTapGesture {
```

Comment:
It needs `.contentShape(Rectangle())` before this line as now it is hard to tap.

Reply:
@PavelHolec
> Binding seems to be ignored:

It's updated when editing ends... I can implement handling each symbol, but is it required? It will require some handling in `shouldChangeCharactersIn` when user deletes char or paste something...

Reply:
We need both, just like the regular InputField (api needs to match):
1) current value (“typing in progress”) binding (to catch “password to short” and similar validation in realtime)
2) and the regular “typing is done”